Project.toml

```
@@ -1,6 +1,6 @@
 name = "ChainRulesTestUtils"
 uuid = "cdddcdb0-9152-4a09-a978-84456f9df70a"
-version = "1.10.1"
+version = "1.10.2"
```

```suggestion
version = "1.11.0"
```
This is not a  patch as  I can't identify a bug this fixed.
I can also imagine wanting to backport to before this

---

Whoops, you're right of course!